client/my-sites/plans/current-plan/index.jsx

```diff
@@ -163,9 +163,9 @@ class CurrentPlan extends Component {
 			translate,
 		} = this.props;
 
-		const currentPlanSlug = selectedSite.plan.product_slug;
+		const currentPlanSlug = selectedSite?.plan.product_slug;
 		const isLoading = this.isLoading();
-		const planTitle = getPlan( currentPlanSlug ).getTitle();
+		const planTitle = getPlan( currentPlanSlug )?.getTitle();
```

Comment:
Maybe we should hide `planFeaturesHeader` if these are not set?

Reply:
I don't really know 😅 This was just a quick fix for something I've noticed; I don't know how it should behave in this case. It's probably super edge case and the real issue might go way deeper (for example, why do we even try to render `CurrentPlan` if we are navigating to the signup flows).